caching/caching_boltdb.go

```diff
@@ -154,6 +188,33 @@ func (c *CachingBolt) GetAppInfo(appGuid string) App {
 	return app
 }
 
+func (c *CachingBolt) alreadyMissed(appGuid string) bool {
+	var d []byte
+
+	c.Appdb.View(func(tx *bolt.Tx) error {
+		b := tx.Bucket([]byte(MISSING_APP_BUCKET))
+		d = b.Get([]byte(appGuid))
+		return nil
+	})
+
+	return bytes.Equal(d, []byte("0"))
```

Comment:
Was trying to be explicit on what we were checking for. Didn't want other values to result in false positives. Changed.